core.js

```diff
@@ -56,6 +56,19 @@ function _check(buffer, headers, options) {
 	return true;
 }
 
+async function _checkSequence(sequence, tokenizer, options) {
+	options = {
+		position: 0,
```

Comment:
0 is a bad default position. Note ` that _check()` may be called in a recursion.

Reply:
Should I default the position to `tokenizer.position` then?

Reply:
Yes, or `undefined`. Or use `await tokenizer.ignore(1350)`.

Reply:
Done in https://github.com/sindresorhus/file-type/pull/323/commits/dc45850aa3aafe47eb41be288e46b54f78bb393b